gdal/ogr/ogrsf_frmts/sosi/ogrsosidriver.cpp

```diff
@@ -121,6 +121,10 @@ void RegisterOGRSOSI() {
     poDriver->SetMetadataItem( GDAL_DCAP_VECTOR, "YES" );
     poDriver->SetMetadataItem( GDAL_DMD_LONGNAME, "Norwegian SOSI Standard" );
     poDriver->SetMetadataItem( GDAL_DMD_HELPTOPIC, "drv_sosi.html" );
+    poDriver->SetMetadataItem( GDAL_DMD_OPENOPTIONLIST,  "<OpenOptionList>"
+    "<Option name='appendFieldsMap' type='string' description='Default is that all rows for equal field names will be appended in a feature, but with this parameter you select what field this should be valid for. With appendFieldsMap=f1&f2, Append will be done for field f1 and f2 using a comma as delimiter.' default=''/>"
```

Comment:
See my previous remark regarding XML escaping of &: `f1&f2` --> `f1&amp;f2`
You can remove default=''